### ISSUE TITLE
Several issues occur while trying other training settings

### ISSUE THREAD

**Issue by AlonzoLeeeooo** [csqiangwen/DeepFillv2_Pytorch#11]
Hi. Thanks for your re-implementation. I got several questions when I was training other settings according to the paper.
The author of DeepFillv2 said that this model can be trained with input channel of 5, which consists of masked image, mask and sketch. I want to train a model to inpaint image with user-guided sketch. But the model cannot converge all the time and the output result was a totally black color image.

Recently I had checked out another repo which is the pytorch re-implementation of the paper Contextual Residual Aggregation by Huawei. The first-stage model is just the same as DeepFillv2. But I notice that the hyper-parameters are not the same. Besides, I noticed that the loss function of the discriminator was hinge loss. But it wasn't implemented in your code. I was quite confused about this, could you please explain why the above issues happen? Thanks a lot!
